blog/2017-08-17-debezium-0-5-2-is-out.adoc

```diff
@@ -0,0 +1,64 @@
+It lets you control how `NUMERIC` and `DECIMAL` columns are represented in change events (either using Kafka's `Decimal` type or as `double`).
+* The MongoDB connector supports the options `database.whitelist` and `database.blacklist` now (https://issues.jboss.org/browse/DBZ-302[DBZ-302])
+* The PostgreSQL connector can deal with array-typed columns as well as with quoted identifiers for tables, schemas etc. (https://issues.jboss.org/browse/DBZ-297[DBZ-297], https://issues.jboss.org/browse/DBZ-298[DBZ-298])
+* The Debezium Docker images run on Red Hat's https://www.openshift.com/[OpenShift] cloud environment (https://issues.jboss.org/browse/DBZ-267[DBZ-267])
+
+Speaking about the Docker images, we've set up _nightly_ tags for the https://hub.docker.com/u/debezium/[Debezium images on Docker Hub],
+allowing you to grab the latest improvements even before an official release has been cut.
+The connector archives are also deployed to the https://oss.sonatype.org/content/repositories/snapshots/io/debezium/[Sonatype OSS Maven repository].
+
+Finally, we've spent some time to extend the documentation on some things not covered before:
+
+* http://debezium.io/docs/configuration/avro/[Avro Serialization] describes how to use the Avro converter - together with the Confluent schema registry - instead of the default JSON converter for serializing change events, resulting in much smaller message sizes;
+The Avro converter itself has also been added Debezium Docker image for Kafka Connect, so you can use it right away
+* http://debezium.io/docs/configuration/topic-routing/[Topic Routing] describes how to use Debezium's `ByLogicalTableRouter` single-message transformation (SMT) for routing the change events from multiple tables into a single topic, which for instance is very useful when working with sharded tables
+
+Please refer to the https://github.com/debezium/debezium/blob/master/CHANGELOG.md#052[changelog] for an overview of all the 19 issues fixed in Debezium 0.5.2.
+
+The following people from the community have sent in pull requests for this release:
+https://github.com/emrul[Emrul Islam], https://github.com/ekreiser[Eric S. Kreiser], https://github.com/xenji[Mario Mueller], https://github.com/mcapitanio[Matteo Capitanio], https://github.com/omarsmak[Omar Al-Safi] and https://github.com/Satyajitv[Satyajit Vegesna].
+
+Thanks a lot to you and everyone else in the community for contributing to Debezium via feature requests, bug reports, discussions and questions!
+
+== What's next
+
+The next version of Debezium will be 0.6 (planned for September).
+This release is planned to bring the upgrade to Kafka 0.11.
+We'll also look into an SMT for transforming the change events emitted by Debezium into a flat representation, which for instance will be very useful in conjunction with the JDBC sink connector.
+In addition we expect to deliver several improvements in terms of correctness of the existing connectors in case of some rare corner cases.
```

Comment:
Yes, good you are saying it. I had some doubts myself, so I'll remove it I think. It's meant for the one you put to "Blocker" btw. and the one related to stumbling across corrupt events.